test-common/src/main/java/org/apache/kafka/common/test/TestKitNodes.java

```diff
@@ -57,8 +57,15 @@ public static class Builder {
         private int numBrokerNodes;
         private int numDisksPerBroker = 1;
         private Map<Integer, Map<String, String>> perServerProperties = Collections.emptyMap();
-        private BootstrapMetadata bootstrapMetadata = BootstrapMetadata.
-            fromVersion(MetadataVersion.latestTesting(), "testkit");
+        private BootstrapMetadata bootstrapMetadata;
+
+        public Builder() {
+            this(BootstrapMetadata.fromVersion(MetadataVersion.latestTesting(), "testkit"));
+        }
+
+        public Builder(BootstrapMetadata bootstrapMetadata) {
```

Comment:
It seems that this new public constructor is not actually being used.

Reply:
I see a usage for `Builder(BootstrapMetadata bootstrapMetadata)` in `KRaftClusterTest`.